pkg/debug/ssh.sh

```diff
@@ -7,4 +7,15 @@ echo -1 > /proc/sys/kernel/perf_event_paranoid
 KEYS=$(find /etc/ssh -name 'ssh_host_*_key')
 [ -z "$KEYS" ] && ssh-keygen -A >/dev/null 2>/dev/null
 
-exec /usr/sbin/sshd -D -e
+
+if [ -f "/config/remote_access_disabled" ]; then
+    # this is picked up by newlogd
+    echo "Remote access disabled, ssh server not started" > /dev/kmsg
+    while true; do
```

Comment:
you can just use `sleep infinity` here, which will block the process indefinitely....

Reply:
are you sure busybox implements all the GNU version options?

Reply:
I checked and it works:
006318ad-dfdc-4a49-87d6-9066cc832b48:~# which sleep
/bin/sleep
006318ad-dfdc-4a49-87d6-9066cc832b48:~# ls -l /bin/sleep
lrwxrwxrwx    1 root     root            12 Sep 12  2022 /bin/sleep -> /bin/busybox
006318ad-dfdc-4a49-87d6-9066cc832b48:~# sleep infinity

Reply:
thanks, changed it.